cvprac/cvp_api.py

```diff
@@ -4070,8 +4103,13 @@ def svc_account_set(self, username, description, roles, status):
                                  'groups': {'values': role_ids},
                                  'key': {'name': username},
                                  'status': status}}
-            url = '/api/v3/services/arista.serviceaccount.v1.AccountConfigService/Set'
-            self.log.debug(f"v7 {url} {payload}")
+            if self.cvp_version_compare('<', 12.0, msg):
+                url = '/api/v3/services/arista.serviceaccount.v1.AccountConfigService/Set'
+                self.log.debug(f"v7 {url} {payload}")
+            else:
+                payload = payload["value"]
+                url = '/api/resources/serviceaccount/v1/AccountConfig'
+                self.log.debug(f"v12 {url} {payload}")
```

Comment:
Is there a reason the conditionals are different here? Can we use the same format of check for >= 12 then fall through to checking >= 7?